src/components/EarnDashboard/components/ProviderDetails/NestedAsset.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Flex, useColorModeValue } from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { Amount } from 'components/Amount/Amount'
+import { NestedListItem } from 'components/List/NestedListItem'
+import { AssetCell } from 'components/StakingVaults/Cells'
+import { RawText, Text } from 'components/Text'
+import type { UnderlyingAssetIdsBalances } from 'state/slices/opportunitiesSlice/utils'
+import { selectAssetById, selectMarketDataById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { OpportunityRowGrid } from './OpportunityTableHeader'
+
+type NestedAssetProps = {
+  assetId: AssetId
+  balances: UnderlyingAssetIdsBalances
+  isClaimableRewards: boolean
+  type: string
+  onClick: () => void
+}
+
+export const NestedAsset: React.FC<NestedAssetProps> = ({
+  assetId,
+  balances,
+  isClaimableRewards,
+  type,
+  onClick,
+}) => {
+  const asset = useAppSelector(state => selectAssetById(state, assetId))
+  const borderColor = useColorModeValue('gray.200', 'gray.700')
+  const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
+  const subText = [<RawText>{type}</RawText>]
+  if (isClaimableRewards) subText.push(<Text color='green.200' translation='common.claimable' />)
+  const subTextJoined = subText.map((element, index) => (
+    <>
+      {index > 0 && <RawText>•</RawText>}
+      {element}
+    </>
+  ))
```

Comment:
Can we do that in a JSX expression instead? The mental model to map over JSX elements is turbo confusing